Python/optimizer_bytecodes.c

```diff
@@ -673,6 +684,8 @@ dummy_func(void) {
             // might be impossible, but bailing is still safe
             ctx->done = true;
         }
+        RELOAD_STACK();
+        res = retval;
```

Comment:
The change in this opcode seems subtle. Why is it wrong to set `res` where it was before? Because it's stolen from `retval`, maybe?

Reply:
If `res` is set before the stack is reloaded from memory, it will be stored to the callee's stack, not the caller's.